gulpfile.js

```diff
@@ -192,33 +218,26 @@ gulp.task('styles-grid', function() {
     .pipe($.size({title: 'styles-grid'}));
 });
 
+// Build with Google's Closure Compiler.
+// To run this step, run "bower install" within MDL to fetch the compiler jar,
+// note that Java 1.7+ is required. This is an optional step.
+gulp.task('closure', function() {
+  return gulp.src(SOURCES)
+    .pipe(closureCompiler({
+      compilerPath: 'bower_components/closure-compiler/compiler.jar',
+      fileName: './dist/mdl.closure.min.js',
```

Comment:
This is tricky. So I've updated the filename - thanks!

However, looks like `gulp-closure-compiler` has [an issue with pipes](https://www.npmjs.com/package/gulp-closure-compiler#implementation-notes). If I specify one, the file just shows up in the top level of MDL anyway. I can't quite work it out, but it looks like Closure has an issue with stdout output, I've left a TODO for now.

If I specify `./dist/material.closure.min.js` it just works and isn't deleted.